src/r2mm/installing/profile_installers/ModLoaderVariantRecord.ts

```diff
@@ -84,6 +84,11 @@ const VARIANTS = {
     SunHaven: [new ModLoaderPackageMapping("BepInEx-BepInExPack", "BepInExPack", PackageLoader.BEPINEX)],
     Wildfrost: [new ModLoaderPackageMapping("BepInEx-BepInExPack_IL2CPP", "BepInExPack", PackageLoader.BEPINEX)],
     ShadowsofDoubt: [new ModLoaderPackageMapping("BepInEx-BepInExPack_IL2CPP", "BepInExPack", PackageLoader.BEPINEX)],
+    WeLoveKatamariRerollRoyalReverie: [new ModLoaderPackageMapping("BepInEx-BepInExPack_IL2CPP", "BepInExPack", PackageLoader.BEPINEX)],
+    Thronefall: [new ModLoaderPackageMapping("BepInEx-BepInExPack", "BepInExPack", PackageLoader.BEPINEX)],
+    Techtonica: [new ModLoaderPackageMapping("BepInEx-BepInExPack", "BepInExPack", PackageLoader.BEPINEX)],
+    GarfieldKartFuriousRacing: [new ModLoaderPackageMapping("BepInEx-BepInExPack", "BepInExPack", PackageLoader.BEPINEX)],
+    WillYouSnail: [new ModLoaderPackageMapping("GodotModding-GodotModLoader", "", PackageLoader.GODOT_ML)],
```

Comment:
Actually this is wrong, the Will You Snail? is a Gamemaker game, not godot. We don't have support for the loader yet, so this can be dropped from the PR.